xgcm/test/test_metrics.py

```diff
@@ -284,7 +308,7 @@ def test_interp_metrics(metric_axes, metric_name):
     grid.set_metrics(metric_axes, metric_name)
     interp_metric = grid._interp_metric(ds.u, metric_axes)
```

Comment:
This one needs to be challenged with the boundary conditions similarly to `interp_like`.